src/encoded/static/components/browse/components/table-commons.js

```diff
@@ -39,7 +39,7 @@ export function defaultColumnBlockRenderFxn(result, columnDefinition, props, wid
         }));
     }
 
-    var value = object.getNestedProperty(result, columnDefinition.field);
+    var value = object.getNestedProperty(result, columnDefinition.field, true);
```

Comment:
Negligible change here (and below in same file) which makes console output less verbose for when encounter `No value` (or `undefined`) for a search table column.